src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterlab/overrides.json

```diff
@@ -50,13 +51,8 @@
                 "rank": 1000,
                 "items": [
                     {
-                        "command": "help:open",
-                        "rank": 1,
-                        "args": {
-                            "url": "/conda-store",
-                            "text": "Environment Management",
-                            "newBrowserTab": true
-                        }
+                        "command": "condastore:open",
```

Comment:
Hi @krassowski thanks for this, could you explain a bit how this works? does this call the extension, or the endpoint for the conda-store webpage as well?

Reply:
It opens conda-store-ui as a tab in jupyterlab main area:

![conda-store](https://github.com/nebari-dev/nebari/assets/5832902/5d9a5910-1b04-429e-9720-9dcfa7e6aeea)